### PR TITLE
Vault install

### PR DESCRIPTION
**Pull request by agentgonzo**
#### Submitter checklist

- [ ] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Description
a `vault-operator` is installed by default when using either of the following two commands:
* `jx install`
* `jx create cluster gke`
In conjunction with either of the two flags:
* `--gitops`
* `--vault` (new flag)

#### Special notes for the reviewer(s)
Please feel free to tell me if this is all wrong as I don't fully understand the codebase yet

### CODE REVIEW

File: pkg/jx/cmd/create_addon_vault.go [hunk 1]
```diff
@@ -77,11 +77,12 @@ func NewCmdCreateAddonVault(f Factory, in terminal.FileReader, out terminal.File
 
 // Run implements the command
 func (o *CreateAddonVaultOptions) Run() error {
-	_, _, err := o.KubeClient()
```
Comment (agentgonzo) [inline pkg/jx/cmd/create_addon_vault.go]: This call to `o.KubeClient()` seemed to be a bit pointless

Comment (ccojocar) [reply pkg/jx/cmd/create_addon_vault.go]: This is just to check if there is a kuberntes connection when running the create addon. It can be ignored.

**Comment by ccojocar** [comment]
We would need to create also a vault after the operator is installed. It's better to send this in a different PR.

**Comment by agentgonzo** [comment]
>We would need to create also a vault after the operator is installed. It's better to send this in a different PR.

Yep, that was my plan!

**Pull request merged**